### ISSUE TITLE
Error: <StoryblokComponent> is not a valid SSR component.

### ISSUE THREAD

**Issue by rogadev** [storyblok/storyblok-svelte#407]
[storyblokurl]: https://www.storyblok.com?utm_source=github.com&utm_medium=issue-template

---

[public repo](https://github.com/rogadev/ITAS268_FinalProject)

## Expected Behavior

After following the setup directions [here](https://www.storyblok.com/tp/add-a-headless-cms-to-svelte-in-5-minutes), at the very last stage I get an error thrown from `Module.validate_component` that suggests the way the tutorial has had me set up this sample project may not, in fact, the way it should be set up.

## Current Behavior

```bash
Error: <StoryblokComponent> is not a valid SSR component. You may need to review your build config to ensure that
    dependencies are compiled, rather than imported as pre-compiled modules. Otherwise you may need to fix a
    <StoryblokComponent>.
    at Module.validate_component (/node_modules/svelte/internal/index.mjs:1869:15)
```

## Steps to Reproduce

1. Follow the documentation/guide in the link provided above to setup a SvelteKit + Storyblok project.
2. I use TypeScript for my SvelteKit project, as I do with all my SvelteKit projects, so be sure to select typescript. Note that this will involve making some edits to the sample components provided in the walk-through.
3. It should throw the same error that you see above.

I've walked the setup back to the beginning and through again. Let me know if there's something I'm missing.

**Comment by nicolette-codes** [comment]
Following the tutorial I'm getting the same error. Only difference is I'm not using typescript.

`Error: <StoryblokComponent> is not a valid SSR component. You may need to review your build config to ensure that dependencies are compiled, rather than imported as pre-compiled modules. Otherwise you may need to fix a <StoryblokComponent>.
    at Module.validate_component (/node_modules/svelte/internal/index.mjs:1869:15)
    at eval (/src/routes/+page.svelte:22:29)
    at Object.$$render (/node_modules/svelte/internal/index.mjs:1892:22)
    at Object.default (root.svelte:42:40)
    at eval (/node_modules/@sveltejs/kit/src/runtime/components/layout.svelte:8:41)
    at Object.$$render (/node_modules/svelte/internal/index.mjs:1892:22)
    at root.svelte:41:39
    at $$render (/node_modules/svelte/internal/index.mjs:1892:22)
    at Object.render (/node_modules/svelte/internal/index.mjs:1900:26)
    at Module.render_response (/node_modules/@sveltejs/kit/src/runtime/server/page/render.js:180:29)`

**Comment by mattpilott** [comment]
Also getting this, have tried switching between dep and devdep but no dice

**Comment by a-luttringer** [comment]
Same here

**Comment by roberto-butti** [comment]
Hi everyone, thank you for all the feedback.
The temporary workaround is to use vite 4.0. (but it is a workaround, not a solution).

We are going to merge the PR 414 and a couple of other upgrades for making the library working also with vite4.1.

**Comment by aaronm-git** [comment]
I am having the same issue. Followed the steps to a T and I am met with a 500 error in your platform.
I have removed the alias pointing to the node and it seems to work this way except that hot reload in your platform, doesn't work after the first hot reload. You need to manually refresh after each change before you can see the changes you make.

**Comment by josefineschaefer** [comment]
Solved with v3.0.0. of storyblok/svelte.
